src/Sulu/Bundle/ResourceBundle/Entity/OperatorTranslation.php

```diff
@@ -0,0 +1,167 @@
+<?php
+
+namespace Sulu\Bundle\ResourceBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * OperatorTranslation
+ */
+class OperatorTranslation
+{
+    /**
+     * @var string
+     */
+    private $name;
+
+    /**
+     * @var string
+     */
+    private $locale;
+
+    /**
+     * @var string
+     */
+    private $shortDescription;
+
+    /**
+     * @var string
+     */
+    private $longDescription;
+
+    /**
+     * @var integer
+     */
+    private $id;
+
+    /**
+     * @var \Sulu\Bundle\ResourceBundle\Entity\Operator
+     */
+    private $operator;
+
+
+    /**
+     * Set name
+     *
+     * @param string $name
+     * @return OperatorTranslation
+     */
+    public function setName($name)
+    {
+        $this->name = $name;
+
+        return $this;
+    }
+
+    /**
+     * Get name
+     *
+     * @return string 
+     */
+    public function getName()
+    {
+        return $this->name;
+    }
+
+    /**
+     * Set locale
+     *
+     * @param string $locale
+     * @return OperatorTranslation
+     */
+    public function setLocale($locale)
+    {
+        $this->locale = $locale;
+
+        return $this;
+    }
+
+    /**
+     * Get locale
+     *
+     * @return string 
+     */
+    public function getLocale()
+    {
+        return $this->locale;
+    }
+
+    /**
+     * Set shortDescription
+     *
+     * @param string $shortDescription
+     * @return OperatorTranslation
+     */
+    public function setShortDescription($shortDescription)
+    {
+        $this->shortDescription = $shortDescription;
+
+        return $this;
+    }
+
+    /**
+     * Get shortDescription
+     *
+     * @return string 
+     */
+    public function getShortDescription()
+    {
+        return $this->shortDescription;
+    }
+
+    /**
+     * Set longDescription
+     *
+     * @param string $longDescription
+     * @return OperatorTranslation
+     */
+    public function setLongDescription($longDescription)
```

Comment:
I don't really mind, but don't we already have something like this already somewhere? If yes we should just use the same names.